frontend/src/plugins/03. analytics.ts

```diff
@@ -20,9 +19,6 @@ export default defineNuxtPlugin(() => {
   }
 
   const uiStore = useUiStore()
-  const featureFlagStore = useFeatureFlagStore()
```

Comment:
The sync is done in `app.vue` and `error.vue` `onMounted`, no need to call it here